cmd/fsi.go

```diff
@@ -20,8 +20,8 @@ func NewFSICommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	}
 
 	link := &cobra.Command{
-		Use:   "link",
-		Short: "link a .qri-ref",
+		Use:   "link DATASET PATH",
```

Comment:
Love it. Reviewing this also had me go lookup the fact that `Use` can be more than one-word long. TIL.